lib/routes/vice/topic.ts

```diff
@@ -0,0 +1,84 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/:topic/:content/:language?',
```

Comment:
```suggestion
    path: '/topic/:topic/:language?',
```
prepanding `/topic` will make this route won't collide with any future routes.